src/metabase/query_processor/middleware/metrics.clj

```diff
@@ -230,6 +231,18 @@
       :else
       expanded-stages)))
 
+(defn- adjust-metric-stages-counting-errors
```

Comment:
What about sg. like `adjust-metric-stages-counting-failure` or `adjust-metric-stages-booking-failure`?
This name suggests me that it will count some errors during processing.

As noted below, I'm not sure this function is needed at all.

Reply:
I also do not like the name or even the existence of this function. See response below, but I will remove this.